cluster-provision/k8s/1.20/provision.sh

```diff
@@ -191,6 +191,11 @@ nmcli connection modify "System eth0" \
    ipv6.addr-gen-mode eui64
 nmcli connection up "System eth0"
 
+dnf install -y centos-release-nfv-openvswitch
+dnf install -y openvswitch2.16
+systemctl enable openvswitch
+systemctl restart openvswitch
```

Comment:
I do not really like the idea of forcing every project to consume OVS when they do not care about it.
But if we must, at least do not let it run there by default. Can you place the service start as part of cluster-up?

We also do not need this for 1.20 anymore.